FEXCore/Source/Interface/Core/ArchHelpers/Arm64Emitter.cpp

```diff
@@ -923,8 +923,8 @@ void Arm64Emitter::PopDynamicRegsAndLR() {
 }
 
 void Arm64Emitter::SpillForPreserveAllABICall(ARMEmitter::Register TmpReg, bool FPRs) {
-  const auto CanUseSVE = EmitterCTX->HostFeatures.SupportsAVX;
-  const auto FPRRegSize = CanUseSVE ? Core::CPUState::XMM_AVX_REG_SIZE : Core::CPUState::XMM_SSE_REG_SIZE;
+  const auto CanUseSVE256 = EmitterCTX->HostFeatures.SupportsSVE256;
+  const auto FPRRegSize = CanUseSVE256 ? Core::CPUState::XMM_AVX_REG_SIZE : Core::CPUState::XMM_SSE_REG_SIZE;
```

Comment:
This should probably changed to `? 32 : 8` but shrug.

Reply:
I'll make a note of it, but I haven't gotten to rewriting the logic of these yet.

Reply:
I just mean, `CanAVX ? AVXSize : SSESize` makes sense, and `Can256 ? 256 : 128` makes sense, but `Can256 ? AVXSize : SSESize` doesn't make sense, so if we're renaming this feature now we should replace the constants with just 256 (32) and 128 (16) to match.

Reply:
Sure, once I return from a nap I will replace it with numeric constants.

Reply:
Switched to numeric constants.